apps/filecopy/src/site/conf/cluster-memory-conf.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
So if someone wants to run this app, he will have to create new config file consolidating all these properties or there is a way to provide multiple properties file to one application?